framework/math/quadratures/angular/product_quadrature.cc

```diff
@@ -165,8 +163,8 @@ AngularQuadratureProdGL::AngularQuadratureProdGL(int Nphemi, bool verbose) : Pro
 
 AngularQuadratureProdGLL::AngularQuadratureProdGLL(int Na, int Np, bool verbose)
 {
-  QuadratureGaussLegendre gl_polar(Np * 2);
-  QuadratureGaussLegendre gl_azimu(Na * 4);
+  GaussLegendreQuadrature gl_polar(Np * 2);
+  GaussLegendreQuadrature gl_azimu(Na * 4);
```

Comment:
I always hated that the Np the user gives is the number of POSITIVE polar angle because it was never referenced somewhere. Can you make sure that this Np has at least a more explicit varname. Same for Na, it is the number of azimuthal angle IN ONE QUADRANT

Reply:
Yes, I'll add that to my refactoring notes for the angular quadratures.